spec/features/user_adds_vendor_spec.rb

```diff
@@ -15,8 +17,18 @@
     member = FactoryGirl.create(:user)
```

Comment:
oh lol I didn't see the collapsed lines, these ARE two separate tests haha. Please ignore!